redis/commands/core.py

```diff
@@ -1762,14 +1761,12 @@ def getex(
         if exat is not None:
             pieces.append("EXAT")
             if isinstance(exat, datetime.datetime):
-                s = int(exat.microsecond / 1000000)
```

Comment:
This can be removed because the maxvalue for `microsecond` is 999999 resulting in `s` being 0 for any valid value.